docs/community/governance.md

```diff
@@ -73,28 +73,77 @@ long experience with both the project and the larger ecosystem. When the core
 developer community (including the SC members) fails to reach such a consensus
 in a reasonable timeframe, the SC is the entity that resolves the issue.
 
-Members of the steering council also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
+Members of the SC also have the "owner" role within the [napari GitHub organization](https://github.com/napari)
 and are ultimately responsible for managing the napari GitHub account, the [@napari_imaging](https://twitter.com/napari_imaging)
 twitter account, the [napari website](http://napari.org), and other similar napari owned resources.
 
-The steering council is currently fixed in size to three members. This number may be increased in
-the future, but will always be an odd number to ensure a simple majority vote outcome
-is always possible. The initial steering council of napari consists of
-
-- [Juan Nunez-Iglesias](https://github.com/jni)
-- [Loic Royer](https://github.com/royerloic)
-- [Nicholas Sofroniew](https://github.com/sofroniewn)
-
-The SC membership is revisited every January. SC members who do
-not actively engage with the SC duties are expected to resign. New members are
-added by nomination by a core developer. Nominees should have demonstrated
+The SC will be no less than three members and no more than five members,
+with a strong preference for an odd number to ensure a simple majority vote
+outcome is always possible, and a preference for five members to ensure a
+diversity of voices. A deadlocked votes of the SC will be postponed until
+there is an odd number of members and another vote can be held. A majority of the
+SC will not be employed by the same entity. One seat on the SC is reserved
+for a member elected by the [Institutional and Funding Partner Advisory Council](institutional-and-funding-partners),
+as detailed below. This member need not be an existing core developer.
+
+The SC membership, including the Institutional and Funding Partner (IFP) seat, is revisited every January.
+SC members who do not actively engage with the SC duties are expected to resign. New members for 
```

Comment:
I think "expected to resign" is slightly unclear. Who among core developers and/or existing SC (if anyone) has the ability to "expel" an SC member who is "not actively engaging"? Would this also be a consensus based decision?

It may be that this is not within scope for this document but it was unclear to me and I just wanted to highlight.

Reply:
That's a valid question - it's language that is already existing in our governance, so not really in scope of the IFP change. For now I'd say if you have that concern you can elevate it to other members of the SC